docs/internal-developers/testing/releases/990.md

```diff
@@ -0,0 +1,314 @@
+
+| Before | After |
+| ------ | ----- |
+|     <img width="772" alt="image" src="https://user-images.githubusercontent.com/14235870/218823036-fe2570cf-fefe-4fc1-be56-4ea628f1bdd5.png">   |    <img width="791" alt="image" src="https://user-images.githubusercontent.com/14235870/218822843-84e82d4c-835d-4076-9125-ec74d1ea9817.png">   |
+
+
+### Enable users to migrate to the blockified Single Product template. ([8324](https://github.com/woocommerce/woocommerce-blocks/pull/8324))
+
+1. Enter the `Single Product` template.
+2. Check that the placeholder description says "This block serves as a placeholder for your WooCommerce Single Product Block. We recommend upgrading to the Single Products block for more features to edit your products visually. Don't worry, you can always revert back.".
+3. Click the `Upgrade to Blockified Single Product Template` button.
+4. See new templates work in the Site Editor and on the front end.
+
+| Before | After |
+| ------ | ----- |
+| <img width="1043" alt="Screenshot 2023-01-30 at 12 41 58" src="https://user-images.githubusercontent.com/186112/215467611-b77d7564-920f-4c05-87af-32e9ed5e6a9b.png"> |<video src=https://user-images.githubusercontent.com/4463174/227149851-235b6685-0955-48e8-91d5-37f2ad7f9b87.mp4/>|
+
+
+### Fix border styles not visible in the editor in Featured Product and Featured Category blocks. ([8838](https://github.com/woocommerce/woocommerce-blocks/pull/8838))
+
+1. Add a Featured Item (Featured Category or Featured Product) block to a page or post.
+2. Select the border controls and add a border style. Add a color and give it some width.
+3. You should see the border you set dynamically display on the featured item.
+4. Now click on the `Unlink` button on the border controls and try setting different values for color and width for each of the border sides (top,right,bottom,left).
+5. Ensure this is working by visually seeing the changes to the featured item.
+6. Save and make sure this is also displaying correctly on the frontend.
+7. Test both `Featured Category` and `Featured Product` blocks.
+
+### Add decoding to product names titles that are in HTML entities. ([8824](https://github.com/woocommerce/woocommerce-blocks/pull/8824))
+
+1. Create a product with special characters in its name (ie: Hat & sticker or Shirt - Green).
+2. In the editor, add a Featured Product, Hand-Picked Products or Reviews by Product block.
+3. Notice in the selector the names of the products with special characters looks correct and not in the HTML entity form.
+
+
+### Fix react 18 error in the editor when using cart/checkout blocks. ([8820](https://github.com/woocommerce/woocommerce-blocks/pull/8820))
+
+- Activate WP 6.2 RC using the WordPress Beta Tester plugin
+- Install [WooCommerce 7.4.1](https://href.li/?https://github.com/woocommerce/woocommerce/releases/tag/7.4.1)
```

Comment:
Maybe add some information here saying that they need to go to Tools > Beta Testing then select the "Bleeding edge" option. They will then need to update their site from Dashboard > Updates